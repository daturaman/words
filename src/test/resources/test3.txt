Â£2.99 15:59 3.1416 -20c
