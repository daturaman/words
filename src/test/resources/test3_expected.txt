Word count = 4
Average word length = 5.000
Number of words of length 4 is 1
Number of words of length 5 is 2
Number of words of length 6 is 1
The most frequently occurring word length is 2, for word lengths of 5